src/view/scripts.inc

```diff
@@ -43,6 +43,14 @@ const SKOSMOS = {
 <!-- Bootstrap -->
 <script src="node_modules/bootstrap/dist/js/bootstrap.bundle.js"></script>
 
+<!-- Enable Bootstrap popovers -->
+<script>
+    var popoverTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="popover"]'))
+    var popoverList = popoverTriggerList.map(function (popoverTriggerEl) {
+      return new bootstrap.Popover(popoverTriggerEl)
+    })
+</script>
+
```

Comment:
This is for the search tips to work as a clickable popover element in Bootstrap. I did not include the search tips box in this PR yet.

Reply:
Let's remove it from this PR as it's not yet needed